OktaJWT.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name             = 'OktaJWT'
-  s.version          = '1.0.0'
+  s.version          = '2.0.0'
```

Comment:
Preferably, I'd like to keep version bump in a separate pull request. Essentially, we've outlined our release process as follows:

1. Have all code changes merged into `master` (usually multiple PRs)
2. Increment the version via a separate release commit. You can see an example [here](https://github.com/okta/okta-ios-jwt/pull/17)
3. Once that is merged, we will push a new tag mapped to `master`
4. Publish to CocoaPods

More publish process information is documented [here](https://oktawiki.atlassian.net/wiki/spaces/PM/pages/241094812/SDK+Publishing+Process).